features/step_definitions/detects_code_duplication_steps.rb

```diff
@@ -1,7 +1,7 @@
 def executing_code_duplication
-  "executing 'flay -d -m 15 .'"
+  "executing 'flay -m 15 .'"
 end
 
 def code_duplication_output
-  command_output_for_project_or_file('flay -d')
+  command_output_for_project_or_file('flay')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.